src/ex4/server/db/migrations/20220619114053-addStatusToItems.cjs

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+module.exports = {
```

Comment:
its worth the effort to keep the convention in the file names. 
in this migration you used "addStatusToItems" while in the other one you wrote "create-items".